packages/lending/package.json

```
@@ -24,10 +24,10 @@
   },
   "dependencies": {
     "@aave/math-utils": "^1.21.0",
-    "@protocolink/api": "^1.0.4",
-    "@protocolink/common": "^0.3.5",
-    "@protocolink/core": "^0.4.5",
-    "@protocolink/logics": "^1.0.3",
+    "@protocolink/api": "^1.0.9",
+    "@protocolink/common": "^0.3.6",
+    "@protocolink/core": "^0.4.6",
+    "@protocolink/logics": "^1.0.8",
```

```suggestion
    "@protocolink/api": "^1.0.10",
    "@protocolink/common": "^0.3.6",
    "@protocolink/core": "^0.4.6",
    "@protocolink/logics": "^1.0.9",
```
These will be the latest version.

---

updated